src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -413,34 +413,34 @@ public void setSnapshot(Tangle tangle, final int index) throws Exception {
     }
 
     /**
-     * This method is the setter for the isSnapshot flag of a transaction.
+     * This method is the setter for the isMilestone flag of a transaction.
      *
      * It gets automatically called by the "Latest Milestone Tracker" and marks transactions that represent a milestone
      * accordingly. It first checks if the value has actually changed and then issues a database update.
      *
      * @param tangle Tangle instance which acts as a database interface
-     * @param isSnapshot true if the transaction is a snapshot and false otherwise
+     * @param isMilestone true if the transaction is a milestone and false otherwise
      * @throws Exception if something goes wrong while saving the changes to the database
      */
-    public void isSnapshot(Tangle tangle, final boolean isSnapshot) throws Exception {
-        if (isSnapshot != transaction.isSnapshot) {
-            transaction.isSnapshot = isSnapshot;
-            update(tangle, "isSnapshot");
+    public void isMilestone(Tangle tangle, final boolean isMilestone) throws Exception {
+        if (isMilestone != transaction.isMilestone) {
+            transaction.isMilestone = isMilestone;
+            update(tangle, "isMilestone");
         }
     }
 
     /**
-     * This method is the getter for the isSnapshot flag of a transaction.
+     * This method is the getter for the isMilestone flag of a transaction.
      *
-     * The isSnapshot flag indicates if the transaction is a coordinator issued milestone. It allows us to differentiate
+     * The isMilestone flag indicates if the transaction is a coordinator issued milestone. It allows us to differentiate
      * the two types of transactions (normal transactions / milestones) very fast and efficiently without issuing
      * further database queries or even full verifications of the signature. If it is set to true one can for example
      * use the snapshotIndex() method to retrieve the corresponding MilestoneViewModel object.
      *
      * @return true if the transaction is a milestone and false otherwise
      */
-    public boolean isSnapshot() {
-        return transaction.isSnapshot;
+    public boolean isMilestone() {
+        return transaction.isMilestone;
```

Comment:
Java convention is that a getter method of a boolean property starts with the prefix `is` and the boolean property itself doesn't have "is" in the name.

It is not something I made up but a java convention I want to follow.
Static code analyzers may take advantage of such conventions for instance.
So please change the property (field name) to milestone